### PR TITLE
fixes wireguard-config issue on container-restart after script-error

### PR DESCRIPTION
**Pull request by bobrossthepainter**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea/
 .local/
+tmp/
```
Comment (dajudge) [inline .gitignore]: Unrelated?

File: helm/templates/deployment.yaml [hunk 2]
```diff
@@ -26,13 +26,17 @@ spec:
...
+          emptyDir: {}
       containers:
         - name: tunnel
           image: dajudge/tunnel-ingress:{{ .Values.tag }}
           volumeMounts:
             - name: ssh-key
               mountPath: /ssh-key/
               readOnly: true
+            - name: tmp
+              mountPath: /tmp/
```
Comment (dajudge) [inline helm/templates/deployment.yaml]: I am wondering if it's wise to use /tmp here. Maybe sth like /var/run/tunnel-ingress/ would be more appropriate? 🤔

I am not really certain but it might even be the case that we wouldn't even need a volume then (maybe /tmp gets special treatment?) 🤔

**Comment by bobrossthepainter** [comment]
@dajudge moved network-setup to init-container. So no need to handle container restarts within main container anymore.

File: .gitignore
Comment (dajudge) [inline .gitignore:1]: Could you please remove this change from the change set?

File: helm/Chart.yaml
Comment (dajudge) [inline helm/Chart.yaml:1]: Could you please remove this change from the change set?

File: scripts/run.sh
Comment (dajudge) [inline scripts/run.sh:1]: This could simply be a `- commmand: [...]` in the yaml, what's you motivation to handle this in a script?

File: scripts/network-init.sh
Comment (dajudge) [inline scripts/network-init.sh:1]: did you mean `network-init-job.sh`?

**Review by dajudge — CHANGES_REQUESTED** [review]

**Review by dajudge — APPROVED** [review]
😍

**Comment by dajudge** [comment]
Hm, this doesn't look like it should be part of this PR? 🤔

**Comment by bobrossthepainter** [comment]
closed in favour of https://github.com/dajudge/tunnel-ingress/pull/2
